ROMFS/px4fmu_common/init.d-posix/airframes/1030_plane

```diff
@@ -11,7 +11,6 @@ param set-default EKF2_MAG_YAWLIM 0
 param set-default FW_LND_AIRSPD_SC 1
 param set-default FW_LND_ANG 8
 
-param set-default FW_L1_PERIOD 12
```

Comment:
TODO: param translation to NPFG_PERIOD

Reply:
@tstastny is there a relation between the NPFG period and the L1 period we could leverage to transform them

Reply:
I thought they are comparable, so what about a 1:1 param translation?

Also FW_L1_R_SLEW_MAX should be translated to FW_PN_R_SLEW_MAX.

Reply:
@KonradRudin yes - they are identical.